docs-website/topics/configure-json-serialization.md

```diff
@@ -0,0 +1,66 @@
+[//]: # (title: JSON serialization overview)
+
+The Kotlin serialization library allows you to easily convert Kotlin objects to JSON and back.
+The [`Json`](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json/) class is the primary tool for this, offering flexibility in how JSON is generated and parsed.
+You can configure `Json` instances to handle specific JSON behaviors or use it without any changes for basic tasks.
```

Comment:
Maybe use words like `use its default instance`? Otherwise it is unclear what to use, because `Json` class does not have a public constructor on its own.